client/src/components/professional/style.css

```diff
@@ -26,14 +26,22 @@
 }
 
 .professionalPaperStyle {
-  background-color: rgba(231,224,215,0.75); 
+  background-color: rgba(231,224,215,0.5); 
+  border-radius: 10px;
   position: relative;
   top: 65vh; 
   bottom: 0;
   width: 65%;
   margin: auto;
 }
 
+.professionalVideo {
+  height: 30vh;
```

Comment:
I think the height and width should be the same value, like 40vh and 40vw. It looks better but doesn't push down the experiences list too much.